### PR TITLE
Updated hash syntax to Ruby 1.9 symbol notation

### PR DESCRIPTION
**Pull request by askl56**

### CODE REVIEW

File: test/support/factories.rb [hunk 1]
```diff
@@ -21,16 +21,16 @@
     sequence(:w_id){ |n| n}
   end
 
-  factory :topic_with_book, :parent=>:topic do |m|
+  factory :topic_with_book, parent::topic do |m|
```
Comment (zdennis) [inline test/support/factories.rb]: I think there needs to be a space above otherwise ruby fails to parse this line.

File: test/models/book.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Book < ActiveRecord::Base
-  belongs_to :topic, :inverse_of=>:books
-  has_many :chapters, :autosave => true, :inverse_of => :book
-  has_many :end_notes, :autosave => true, :inverse_of => :book
+  belongs_to :topic, inverse_of::books
```
Comment (zdennis) [inline test/models/book.rb]: Space needed on this line.

File: test/models/chapter.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 class Chapter < ActiveRecord::Base
-  belongs_to :book, :inverse_of=>:chapters
-  validates :title, :presence => true
+  belongs_to :book, inverse_of::chapters
```
Comment (zdennis) [inline test/models/chapter.rb]: Space needed.

File: test/models/end_note.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
 class EndNote < ActiveRecord::Base
-  belongs_to :book, :inverse_of=>:end_notes
-  validates :note, :presence => true
+  belongs_to :book, inverse_of::end_notes
```
Comment (zdennis) [inline test/models/end_note.rb]: Space needed.

File: test/models/topic.rb [hunk 5]
```diff
@@ -2,8 +2,8 @@ class Topic < ActiveRecord::Base
   validates_presence_of :author_name
   validates :title, numericality: { only_integer: true }, on: :context_test
 
-  has_many :books, :autosave=>true, :inverse_of=>:topic
-  belongs_to :parent, :class_name => "Topic"
+  has_many :books, autosave:true, inverse_of::topic
```
Comment (zdennis) [inline test/models/topic.rb]: Spaced needed.

**Comment by zdennis** [comment]
Thanks @askl56 for the PR. I think some of the test files are missing spaces so things like `:foo => :bar` got changed to `foo::bar`.

**Comment by sferik** [comment]
Sorry for all the noise here. In an effort to debug tests failing on Travis CI, I had to bisect and force push this [pull request](https://github.com/zdennis/activerecord-import/pull/250) half a dozen times. The good news is, this issue can now be closed.

**Comment by zdennis** [comment]
Thanks @askl56 and @sferik. Closed this since #250 covers this.

**Pull request closed without merging**